src/components/styles/Theme.tsx

```diff
@@ -332,15 +343,26 @@ export const zopaTheme: AppTheme = {
     iconBackgroundColor: colors.greyLighter,
     searchInput: {
       boxShadow: `0 0 4px 0 ${colors.brand}`,
-      borderRadiusInput: `8px 8px 0 0`,
-      borderRadiusOptions: `0 0 8px 8px`,
+      borderRadius: `8px 8px 0 0`,
       borderColor: colors.brand,
       customIcon: false,
-      optionHover: {
-        color: colors.white,
-        backgroundColor: colors.brand,
+      options: {
+        borderRadius: `0 0 8px 8px`,
```

Comment:
Are these 8s meaningful in our scale anywhere, wondering if there are some values we can pull from rather than hardcoding the '8px' in there? If not then I'm happy for this to merge

Reply:
potentially we could tie it up to `spacing` e.g. `spacing.2` but as it is hardcoded across multiple components, maybe it makes sense to have a specific task to change it in multiple places so it is consistent?

Reply:
Yeah understood, ok lets go with this and then clean up later.